src/DropdownMenu/Menu.tsx

```diff
@@ -94,7 +94,7 @@ export default class Menu extends React.PureComponent<MenuProps, MenuState> {
         <FlexView grow shrink className='menu-item-title'>
           {option.title}
         </FlexView>
-        <FlexView grow shrink={false} className='menu-item-metadata' hAlignContent='right'>
+        <FlexView shrink={false} className='menu-item-metadata' hAlignContent='right'>
```

Comment:
@FrancescoCioria double check this makes sense. It's potentially breaking.

Reply:
I tested and it seems to work 👍